src/views/settings/Storage/ReplayStorage.vue

```diff
@@ -60,9 +63,6 @@ export default {
           },
           actions: {
             formatterArgs: {
-              onUpdate: function({ row }) {
-                this.$router.push({ name: 'ReplayStorageUpdate', params: { id: row.id }, query: { type: row.type.value }})
-              },
               canUpdate: function({ row }) {
                 return (
                   row.name !== 'default' && row.name !== 'null' && vm.$hasPerm('terminal.change_replaystorage')
```

Comment:
In summary:

- There is unnecessary parentheses around the `ref` property of `<ListTable>`
- The variable name between double quotes should be in camelCase format instead of snake_case

Here's my suggestion to make it cleaner and more consistent:
    @@ -1,5 +1,7 @@
 <template>
 
   <ListTable
             ref="ListTable"
+        :replay-storage-table-ref="'ListTable'"
              :header-actions="(action) => ({ canCreate: $hasPerm('terminal.create_replaystorage'), ...action })?"
+          :list-action="{...action}"
+          :table-config="replayTableConfig"
+            :drawer-component={() => import('./ReplayStorageCreateUpdate.vue')}
             
@ -15,9 +18,8 @@

@end
 </template>
 <script>

```